### PR TITLE
update generating tab group id.

### PR DESCRIPTION
**Pull request by vwxyzh**
#1659

### CODE REVIEW

File: src/Microsoft.DocAsCode.Dfm/DfmEngineBuilder.cs [hunk 1]
```diff
@@ -93,6 +93,35 @@ public DfmEngineBuilder(Options options, string baseDir, string templateDir, IRe
...
+                            return null;
+                        }
+                        else
+                        {
+                            return new DfmTabGroupBlockToken(token.Rule, token.Context, value, token.Items, token.ActiveTabIndex, token.SourceInfo);
+                        }
+                    }
+                    else
+                    {
+                        dict[token.Id] = index + 1;
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.Dfm/DfmEngineBuilder.cs]: >token.Id [](start = 29, length = 8)

value?

**Comment by superyyrrzz** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Dfm/Aggregators/TabGroupAggregator.cs [hunk 2]
```diff
@@ -67,11 +67,12 @@ protected override bool AggregateCore(MarkdownHeadingBlockToken headToken, IMark
                 md += terminator.SourceInfo.Markdown;
                 offset++;
             }
+            var groupId = (items[0].SourceInfo.File ?? string.Empty).GetMd5String().Replace("/", "-").Remove(10);
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Dfm/Aggregators/TabGroupAggregator.cs]: >Remove(10) [](start = 102, length = 10)

why remove 10?

**Comment by hellosnow** [comment]
:shipit:

**Comment by vicancy** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Dfm/Aggregators/TabGroupAggregator.cs (hunk 2 above)
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Dfm/Aggregators/TabGroupAggregator.cs]: just short the id.

**Pull request merged**